shell/platform/windows/flutter_windows_winuwp.cc

```diff
@@ -25,9 +28,25 @@ static flutter::FlutterWindowsEngine* EngineFromHandle(
   return reinterpret_cast<flutter::FlutterWindowsEngine*>(ref);
 }
 
+std::vector<std::string> split(std::string const s) {
```

Comment:
You can do this more simply with stringstream; see https://github.com/flutter/engine/blob/83732f262cc1bef0634530f22dc6ec151ab9f6df/shell/common/switches.cc#L387 or https://github.com/flutter/engine/blob/9f207a3fc6db6f121d2f247860f7f634e8438295/shell/platform/windows/system_utils_win32.cc#L66